pkg/oc/cli/admin/migrate/migrator.go

```diff
@@ -228,12 +228,12 @@ func (o *ResourceOptions) Complete(f kcmdutil.Factory, c *cobra.Command) error {
 		if resourceNames.Has(s) {
 			continue
 		}
-		if s != "*" {
+		if !strings.HasPrefix(s, "*") {
 			resourceNames.Insert(s)
 			break
 		}
 
```

Comment:
In the process of working on #21071 I realized that once you get past this point you no longer do any more iterations of the loop (since it `break`s at the end).  While that is OK for `*`, it does not make sense for `*.foo,*.bar` or similar.  If we ever go this route we may need to refactor this whole block.